StackableGems/mod.json

```
@@ -1,15 +1,31 @@
 {
   "name": "Stackable Gems",
-  "description": "Adds another variation of gems that can stack together. Transmute a gem in the Cube to convert it between the two variants. Based on https://www.nexusmods.com/diablo2resurrected/mods/80.",
+  "description": "Adds another variation of gems that can stack together. Transmute a gem in the Cube to convert it between the two variants. Easy upgrade stack of gems. Based on https://www.nexusmods.com/diablo2resurrected/mods/80.",
   "author": "olegbl",
   "website": "https://github.com/olegbl/d2rmm.mods",
-  "version": "1.4",
+  "version": "1.5",
   "config": [
+    {
+      "id": "maxInStack",
+      "type": "number",
+      "name": "Maximum items in stack",
+      "description": "Maximum items in stack. ",
+      "defaultValue": 10,
```

We'll still need to keep the default at 500. Otherwise, if somebody updates from an old version and doesn't check the preferences, they could end up deleting some of their items.

```suggestion
      "defaultValue": 500,
```